### PR TITLE
CPDRP-308: sign out page

### PR DESCRIPTION
**Pull request by ethanmills**
### Context
Add a sign out page

### Changes proposed in this pull request
image

### Review Checks
- [x] All pages have automated accessibility checks via cypress
- [x] All pages have visual tests via cypress + percy
- [x] All `School` queries are correctly scoped - `eligible` when they need to be

### How can I view this in a review app?
Sign in as admin@example.com
Sign out

### CODE REVIEW

File: spec/cypress/integration/accessibility.js [hunk 1]
```diff
@@ -48,5 +48,9 @@ describe("Accessibility", () => {
     cy.get('[action="/users/sign_in_with_token"] [name="commit"]').click();
     cy.get("h1").should("contain", "User dashboard");
     cy.checkA11y();
+
+    cy.logout();
+    cy.checkA11y();
+    cy.percySnapshot("Logout page");
```
Comment (kicferk1) [inline spec/cypress/integration/accessibility.js:54]: I wonder if we should do that in cucumber

**Review by kicferk1 — APPROVED** [review]
Code looks good, but I think a cucumber test might be a better place than cypress test for checking percy and accessibility on logout page.

File: spec/cypress/integration/accessibility.js (hunk 1 above)
Comment (ethanmills) [reply spec/cypress/integration/accessibility.js:54]: I wanted to keep it with the tests for the login page

Comment (ethanmills) [reply spec/cypress/integration/accessibility.js:54]: (and didn't want to rewrite those in cucumber)

**Review by callumacrae — APPROVED** [review]

**Pull request merged**